service/controller/chart/v1/resource/release/create_test.go

```diff
@@ -0,0 +1,99 @@
+package release
+
+import (
+	"context"
+	"testing"
+
+	"github.com/giantswarm/apiextensions/pkg/apis/application/v1alpha1"
+	"github.com/giantswarm/helmclient/helmclienttest"
+	"github.com/giantswarm/micrologger/microloggertest"
+	"github.com/spf13/afero"
+	"k8s.io/client-go/kubernetes/fake"
+)
+
```

Comment:
Test is based on the chartconfig resource. I just updated the types and tidied up a bit.